src/components/mortgage-investment-compare/index.js

```diff
@@ -152,12 +166,14 @@ class MortgageInvestmentCompare extends React.Component {
       }
     })
 
-    return {
+    const result = {
```

Comment:
sug: This could be directly returned

Reply:
I like to name objects. I know it's obvious because the name of the function is `getResult` but for some reason when you first created this function I didn't think the function actually returned the whole result object in the state